androidApp/src/main/java/dev/johnoreilly/confetti/ui/ConfettiScaffold.kt

```diff
@@ -60,10 +64,32 @@ fun ConfettiScaffold(
     content: @Composable (ScaffoldState) -> Unit,
 ) {
     val authentication = koinInject<Authentication>()
+    val wearSettingSync = koinInject<WearSettingsSync>()
     fun signOut() {
         authentication.signOut()
         onSignOut()
     }
+
+    val coroutineScope = rememberCoroutineScope()
+
+    val wearSettingNodeState =
+        wearSettingSync.wearNodes.collectAsStateWithLifecycle(emptyList()).value
+    val wearSettingsUIState = WearUiState(wearSettingNodeState)
+
+    fun installOnWear() {
+        coroutineScope.launch {
+            wearSettingNodeState.filter { !it.isAppInstalled }.forEach {
+                wearSettingSync.installOnWearNode(it.id)
+            }
+        }
+    }
+
+    fun updateWearTheme() {
+        coroutineScope.launch {
+            wearSettingSync.updateWearTheme()
+        }
+    }
```

Comment:
This I'm not 100% sure about 🤔

Reply:
My hunch is that this should run in some global scope, not the composable scope. If you move out of the composable, you probably don't want to cancel the installation. I'd go with something like this directly inside `WearSettingsSync` but @yschimke will know better.

```kotlin
    private var job: Job? = null

    // public non-suspend function to be called from button callbacks, etc...
    fun installOnWearNode(nodeId: String) {
        if (job != null) {
            // already installing, skip
            return
        }
        
        // coroutineScope at this point is a app-global scope
        job = coroutineScope.launch {
            installOnWearNodeInternal(nodeId)
            
            job = null
        }
        
    }
    private suspend fun installOnWearNodeInternal(nodeId: String) { ... }
```

Reply:
I guess three options.

a) the composable is responsible for this completing. So we should model the inprogress state even a spinner.
b) since it's going to be almost immediate, we actually block on it completing but don't show anything.
c) We move it up to a wider scope.  We just lost view model scope, so maybe the global scope for now?

Reply:
hm, maybe I can provide the activity lifecycle scope and use that? I feel like using globalScope is a bit heavy and according to the documentation, easy to accidentally create resource or memory leaks :)

Reply:
I liking providing a small UI or blocking: gives feedback to the user that something is happening but this is more involved and can be done in a separate PR. 

Re: global scope, I had [this one](https://github.com/joreilly/Confetti/blob/ba7b222b5bcc211a15b4bad7ab8ce1a92b80d6df/androidApp/src/main/java/dev/johnoreilly/confetti/wear/WearSettingsSync.kt#L29) in mind but activityScope works too!

Reply:
Aha sorry I misunderstood that. I think that can be used :)